dotnet/src/SemanticKernel.Abstractions/Functions/KernelFunction.cs

```diff
@@ -87,19 +91,58 @@ internal KernelFunction(string name, string description, IEnumerable<AIRequestSe
         using var activity = s_activitySource.StartActivity(this.Name);
         ILogger logger = kernel.LoggerFactory.CreateLogger(this.Name);
 
-        logger.LogInformation("Function invoking.");
+        if (logger.IsEnabled(LogLevel.Trace))
+        {
+            logger.LogTrace("Function {Name} invoking.", this.Name);
```

Comment:
Isn't the function name here baked into the logger? Why does the name also need to be part of the message?